6_outro_gif_frames.yml

```diff
@@ -22,14 +22,14 @@ targets:
       view_fun,
       basemap_fun,
       ocean_name_fun,
-      precip_raster_fun_a_20180919_09,
+      precip_raster_fun_a_20190715_12,
```

Comment:
There are a bunch of little changes likes these in the PR that I'd have to run code to evaluate. Have you run this code this spring, Lindsay, such that you're at least confident that these style/data changes don't break the pipeline or ruin the gif?

Reply:
TBH, I'm not really sure where some of these changes came from (they are things that should only be in the `Makerspace-example` branch, which picked a smaller storm with no hurricane. I thought it wouldn't be the end of the world to just leave them rather than revert, but some of your comments below make me want to revert those. 

Those changes are mostly reverting https://github.com/USGS-VIZLAB/vizstorm-GIF/pull/218/commits/376ae2e03c649e7491d5a7aed704fc65ed8bf7da and https://github.com/USGS-VIZLAB/vizstorm-GIF/pull/218/commits/ab2af0a5e264ef9a0d587c5b5a2ea4ac25635ae7 and the change this comment is referring to.